tools/syz-repro/repro.go

```diff
@@ -22,6 +22,8 @@ var (
 	flagConfig = flag.String("config", "", "manager configuration file (manager.cfg)")
 	flagCount  = flag.Int("count", 0, "number of VMs to use (overrides config count param)")
 	flagDebug  = flag.Bool("debug", false, "print debug output")
+	flagOutput = flag.String("output", "./output.txt", "output description file (output.txt)")
+	flagCRepro = flag.String("crepro", "./repro.c", "output c file (repro.c)")
```

Comment:
It's better to accept only a single dir flags and all outputs are written there with fixed names. For a repro we generally write syz repro, C repro, log, report, description and tag. Making user to change that many flags to write a specified dir is not graceful. Also if we unify writing code with manager, it won't change behavior.

Reply:
My intention was to give users an option of specifying output file with default values. But I can change this to one output directory arg and default to saving `repro.txt` and `repro.c` in a given output directory.